addons/edi/models/edi_document.py

```diff
@@ -427,14 +433,11 @@ def commit_execute_progress(self, log_message):
         # Restore the original records to write in the main transcation
         self.env.all.towrite = towrite
 
+    def commit_execute_progress(self, log_message):
```

Comment:
Q: Is this function needed anymore? Or can we just call `create_or_update_edi_progress` directly?